plugins-ui/src/__ tests __/policy/PolicyProvider.test.tsx

```
@@ -6,68 +6,90 @@ import {
+const mockPlugin = {
+  id: "1",
+  type: "type",
+  title: "Plugin title",
+  description: "Plugin description",
+  metadata: {},
+  server_endpoint: "endpoint",
+  pricing_id: "pricingId",
+};
+
+vi.mock("react-router-dom", async (importOriginal) => {
+  const actual = (await importOriginal()) as {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)